README.md

```diff
@@ -1,87 +1,90 @@
 # VisCy
 
-VisCy is a deep learning pipeline for training and deploying computer vision models for image-based phenotyping at single-cell resolution.
-
-The following methods are being developed:
+VisCy (abbreviation of `vision` and `cyto`) is a deep learning pipeline for training and deploying computer vision models for image-based phenotyping at single-cell resolution.
 
+This repository provides a pipeline for the following.
 - Image translation
   - Robust virtual staining of landmark organelles
 - Image classification
   - Supervised learning of of cell state (e.g. state of infection)
 - Image representation learning
   - Self-supervised learning of the cell state and organelle phenotypes
 
-<div style="border: 2px solid orange; padding: 10px; border-radius: 5px; background-color: #fff8e1;">
+<div style="border: 2px solid orange; padding: 10px; border-radius: 5px">
   <strong>Note:</strong><br>
-  VisCy is currently considered alpha software and is under active development. Frequent breaking changes are expected.
+  VisCy has been extensively tested for image translation task. The code for other tasks is under active development. Frequent breaking changes are expected in the main branch as we unify the codebase for above tasks. If you are looking for a well-tested version for virtual staining, please use release 0.2.1 from PyPI.
 </div>
```

Comment:
This style doesn't seem to do anything on GitHub.

Reply:
I also realized this. @mattersoflight pointed out that this was not visible if someone is using the darkmode. I'm swapping this to show as a blockquote.